### ISSUE TITLE
This extension violates the Chrome Web Store guidelines.

### ISSUE THREAD

**Issue by Dtrieb** [notronik/pace4chrome#2]
This extension violates the Chrome Web Store guidelines.
So its not in the Web Store anymore.

**Comment by nvgrw** [comment]
Yeah they changed some policy and I haven't had the time to comply with whatever changed
